modal_docs/mdmd/signatures.py

```diff
@@ -70,4 +71,7 @@ def get_signature(name, callable) -> str:
         definition_source = definition_source.replace("asynccontextmanager", "contextmanager")
         definition_source = definition_source.replace("AsyncIterator", "Iterator")
 
+    # remove any synchronicity-internal decorators
+    definition_source, _ = re.subn(r"^\s*@synchronizer\..*\n", "", definition_source)
```

Comment:
Somewhat unrelated, but I noticed that our reference docs keep the synchronicity-decorators on presented functions which is a bit ugly

Reply:
I'm not sure - many are necessary to know how to use the functions, e.g.  `@classmethod`, `@contextmanager`. Might have to browse through the reference docs to see if include or exclude lists would work best...